dts/bindings/pwm/xlnx,xps-timer-1.00.a-pwm.yaml

```diff
@@ -0,0 +1,24 @@
+description: Xilinx AXI Timer IP node (PWM controller)
+
+compatible: "xlnx,xps-timer-1.00.a-pwm"
+
+include: [ "xlnx,xps-timer-1.00.a.yaml", pwm-controller.yaml ]
+
+properties:
```

Comment:
Will anybody who uses this know what these mean?

Reply:
@pabigot Thank you for reviewing. The properties map 1:1 to the configuration options for the IP core within Xilinx Vivado, so my guess would be "yes". Furthermore, the properties are documented in `dts/bindings/rtc/xlnx,xps-timer-1.00.a.yaml`.